packaging/standalone/src/main/distribution/shell-scripts/bin/neo4j-shell.m4

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+set -o errexit -o nounset -o pipefail
+[[ "${TRACE:-}" ]] && set -o xtrace
```

Comment:
These two lines can go in the shared file. They should be the same for every script.